stable/database/README.md

```diff
@@ -303,7 +309,20 @@ Any file located in `database.configFilesPath` can be replaced; the YAML key cor
 | ----- | ----------- | ------ |
 | `nuodb.config` | [NuoDB database options][6] | `nil` |
 
-#### database.serviceSuffix
+#### restore.*
+
+The following tables list the configurable parameters of the restore chart and their default values.
+
+| Parameter | Description | Default |
+| ----- | ----------- | ------ |
+| `restore.type` | What type of restore to perform: [ "database" | "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
```

Comment:
does the database have to be `RUNNING`? Can you restore a single archive with a DB that is down? I know that it virtually becomes a `database` restore, but will the flag work?

Reply:
The database doesn't have to be running, however, the admin layer should. `nuorestore` just sets a flag in KV store which indicates that restore needs to be performed on the next database restart.